### PR TITLE
Bug 1145966 - When attempting to view a non-existent repo with &revision=SHA, display the correct error message

### PR DESCRIPTION
**Pull request by mandaltapesh**
Hi all ..Respected admins and all my fellow group members,
    this is an attempt to improve the error message display functionality while attempting to view a non-existent repo and revision..there are 2 changes in jobs.html..i have received immense amount of help on the way from edmorley..i have tested the changes in my PC.(ubuntu 14.04 64bit ,firefox 33.0).
and they were found to be satisfactory.
    i humbly request edmorley to kindly ignore the indentation diffs..i am not being able to fix that..
yours sincerely
Tapesh Mandal

[Review on Reviewable](https://reviewable.io/reviews/mozilla/treeherder-ui/475)

### CODE REVIEW

File: webapp/app/partials/main/jobs.html [hunk 1]
```diff
@@ -52,13 +52,13 @@
...
   <span ng-init="revision=getSearchParamValue('revision')">
-      <span>Waiting for a push with revision <strong>{{revision}}</strong></span>
-      <a href="{{currentRepo.url}}/pushloghtml?changeset={{revision}}"
-         target="_blank"
-         title="open revision {{revision}} on {{currentRepo.url}}">(view pushlog)</a>
+     <span>Waiting for a push with revision <strong>{{revision}}</strong></span>
+     <a href="{{currentRepo.url}}/pushloghtml?changeset={{revision}}"
+        target="_blank"
+        title="open revision {{revision}} on {{currentRepo.url}}">(view pushlog)</a>
     <span class="fa fa-refresh fa-spin"></span>
```
Comment (edmorley) [inline webapp/app/partials/main/jobs.html]: There's some extra indentation here - if you are struggling to get this back to how it was - copy the original lines from the existing file back over your copy, eg:
https://github.com/mozilla/treeherder-ui/blob/master/webapp/app/partials/main/jobs.html#L58

**Comment by edmorley** [comment]
This works well when I tested locally, so I just need you to:
1) Fix the indentation mentioned above
2) Rebase this against latest master
3) Squash your commits into just one commit, and use a commit message that is more descriptive of the changes being made, and includes the bug number in the title (see previous commits for examples).

For links to guides to rebasing, see https://ateam-bootcamp.readthedocs.org/en/latest/reference/git_github.html or google "rebasing pull request" or "squashing pull request".

Well done - almost there! :-)

**Comment by mandaltapesh** [comment]
when i try to squash 8a1c392 into  3a66caf it says :
fatal: Needed a single revision
Invalid commit name: 3a66caf

**Comment by edmorley** [comment]
What command were you using? An error without the command that led to it, is hard to diagnose :-)

I'd suggest following a guide like this one http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit (I found that one by googling "squash commits pull request").

**Comment by mandaltapesh** [comment]
git rebase -i HEAD~2
pick 8a1c392
squash 3a66caf
reword ...  renaming commit messages
reword...

**Comment by mandaltapesh** [comment]
sir 
kindly check this squashed it with previous bug commit..as git log shows not more than 1 commit

regards,
Tapesh Mandal

**Comment by edmorley** [comment]
Replaced by https://github.com/mozilla/treeherder-ui/pull/479

**Pull request closed without merging**